elections/end-user-electors.md

```diff
@@ -0,0 +1,16 @@
+# End User Electors
+
+This document lists each approved elector capable of casting a vote for candidates for the End User Seat on the TSC.
+Each entry in this file must contain:
+- The name of the project or business
+- The Hedera account of the elector
```

Comment:
Can we split this in the 2 columns "Network" and "Account"?

"Network" must contain the name of a public Hiero based network. At the start that will be "Hedera Mainnet" for 100%
"Account" contains the account of the person in the given network